src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/MessageProcessorTest.java

```diff
@@ -194,7 +194,7 @@ void should_update_hold_until_and_retry_count_for_ready_messages_when_retryable_
                     "classpath:sql/insert_case_event_messages_for_processing_ready_msgs.sql"})
```

Comment:
do we need to also cover setting message_state to Ready when retryable_error?

Reply:
Ben I have not modified existing logic setting message_state to Ready, have only classified what error are retryable and unprocessable.